src/black_primer/primer.json

```
@@ -30,13 +30,16 @@
       "py_versions": ["all"]
     },
     "django": {
-      "disabled_reason": "black --check --diff returned 123 on tests_syntax_error.py",
-      "disabled": true,
-      "cli_arguments": ["--experimental-string-processing"],
+      "cli_arguments": [
+        "--experimental-string-processing",
+        "--skip-string-normalization",
+        "--extend-exclude",
+        "/((docs|scripts)/|django/forms/models.py|tests/gis_tests/test_spatialrefsys.py|tests/test_runner_apps/tagged/tests_syntax_error.py)"
+      ],
       "expect_formatting_changes": true,
       "git_clone_url": "https://github.com/django/django.git",
       "long_checkout": false,
-      "py_versions": ["all"]
+      "py_versions": ["3.8", "3.9"]
```

Here is where PEP440 version specifiers could be nice:
```suggestion
      "py_versions": [">=3.8"]
```

---

The development environment already mandates `packaging` as required due to Sphinx, tox, and pytest. We could lock away primer under an extra to avoid the extra dependency for production installs.